torchao/kernel/intmm_triton.py

```diff
@@ -0,0 +1,349 @@
+import torch
+from torchao.kernel.autotuner import get_best_config_fn
+from torch._higher_order_ops.out_dtype import out_dtype
+from torch._dynamo import is_compiling as dynamo_is_compiling
+
+import triton
+import triton.language as tl
+import itertools
+import os
+int8_powers_of_two = [32, 64, 128, 256]
+int8_mm_kernel_configs = sum([
+    # "BLOCK_M", "BLOCK_N", "BLOCK_K", "num_stages", "num_warps"
+        [
+            (i, j, k, 1, 1),
+
+            (i, j, k, 1, 2),
+            (i, j, k, 2, 2),
+
+            (i, j, k, 1, 4),
+            (i, j, k, 2, 4),
+            (i, j, k, 3, 4),
+            (i, j, k, 4, 4),
+
+            (i, j, k, 1, 8),
+            (i, j, k, 2, 8),
+            (i, j, k, 3, 8),
+            (i, j, k, 4, 8),
+            (i, j, k, 5, 8),
+            (i, j, k, 6, 8),
+            (i, j, k, 7, 8),
+            (i, j, k, 8, 8),
+        ] for 
+        (i, j, k) in 
+        itertools.product(int8_powers_of_two,
+                          int8_powers_of_two,
+                          int8_powers_of_two)], [])
+
+# int8_mm_kernel_configs = [
```

Comment:
delete?

Reply:
I wanted to leave these as reference from core. I'll add a comment.